### PR TITLE
Remove pika set log level

### PR DESCRIPTION
**Pull request by tej99**
Removed setting log level for pika logger as it was causing problems picking up logs in splunk.

## Checklist
  - [ ] CHANGELOG.md updated? (if required)

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -14,7 +14,6 @@ def run():
     logging.basicConfig(format=app.settings.LOGGING_FORMAT,
                         datefmt="%Y-%m-%dT%H:%M:%S",
                         level=app.settings.LOGGING_LEVEL)
-    logging.getLogger("pika").setLevel(logging.INFO)
```
Comment (necrophonic) [inline app/main.py]: What happens if we don't set it at all? Does it default to the most verbose level or does it get switched off? I thought it wouldn't log anything unless we explicitly set the logger?

Comment (tej99) [inline app/main.py]: Defaults to the most verbose level

Comment (warrenbailey) [reply app/main.py]: I think this should be configurable. Can you create another setting in app.settings?

Comment (JamesGardiner) [reply app/main.py]: :+1: for configuring this. We may want it at different levels dependent on the deployed environment.

**Pull request closed without merging**